src/components/TextArea.jsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Grid, TextField } from '@mui/material';
+import TextareaAutosize from '@mui/material/TextareaAutosize';
+
+export default function TextArea() {
+  return (
+      <Grid container spacing={1}>
+        <Grid item xs={12} md={12}>
+          <TextareaAutosize
```

Comment:
Please use TextField with multiline so it will be more stylish 
https://mui.com/components/text-fields/#multiline